skiros2_skill/src/skiros2_skill/ros/skill_manager.py

```diff
@@ -387,6 +389,14 @@ def __init__(self):
     def _set_debug_cb(self, msg):
         self.publish_runtime_parameters = msg.data
 
+    def _update_skills_cb(self, _):
+        self.sm.skills.clear()
```

Comment:
This is clearing skills form the SkillManager, but it's not clearing skills from the World Model